src/lib/rpmostree-package.c

```diff
@@ -32,11 +32,11 @@
 
 #include <string.h>
 #include <stdlib.h>
-#include <rpm/rpmver.h>
 #include "libglnx.h"
 
 #include "rpmostree-shlib-ipc-private.h"
 #include "rpmostree-package-priv.h"
+#include "rpmver-private.h"
```

Comment:
Hmm, can we make this a build opt instead? I.e. we auto-detect if the API is present and only if not do we pull this in. (See https://github.com/coreos/rpm-ostree/pull/824 for example autoconf goop.)